cypress/integration/test.js

```diff
@@ -4,12 +4,18 @@ describe('App', () => {
     cy.visit('http://localhost:3000');
   })
 
-  it('Should see a header and home button on page load', () => {
+  it('Should see a header, form and home button on page load', () => {
     cy.get('header h1').should('have.text', 'RancidTomatillos');
     cy.get('header button').should('have.text', 'HOME');
+    cy.get('form').and('be.visible');
   });
 
-  it('Should be able to click a movie and direct it to a new URL to see movie details and click on the home button to go back', () => {
+  it('Should have a drop down menu and be able to select each options', () => {
+    cy.get('select').select('Date');
+    cy.get('select').select('Freshness');
+  });
```

Comment:
I think if we have time it could be nice to come back and add a test to make sure the sorting is right. I'm not comfortable enough with Cypress to suggest specific code but there might be a way to test if the first item in the `MovieList` is a specific movie. I think this more than good enough if we don't get to it though.

Reply:
Also I'm realizing we could do the same for the search/filter option. Something like searching for `'Title 1'` and then making sure that `cy.get('.poster')` has a length of 1.

Reply:
awesome I added in a length for this.